src/layers/HeatmapCompositeTextLayer.js

```diff
@@ -25,7 +25,7 @@ export default class HeatmapCompositeTextLayer extends CompositeLayer {
         getText: d => d[1],
         getPosition: d => [matrixLeft + ((d[0] + 0.5) / width) * matrixWidth, axisLabelTop],
         getTextAnchor: 'start',
-        getColor: THEME_TO_TEXT_COLOR[theme],
+        getColor: () => THEME_TO_TEXT_COLOR[theme],
```

Comment:
These changes were to fix an issue I was seeing with the test runner throwing a DeckGL error about these not being functions.